html/pages/edituser.inc.php

```diff
@@ -432,7 +436,7 @@
             }//end if !empty($users_details)
         }//end if
     } else {
-        $user_list = LegacyAuth::get()->getUserlist();
+        $userlist = User::all();
```

Comment:
You need to use `User::thisAuth()->all();` to exclude users from inactive auths.